tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchCMISTests.java

```diff
@@ -147,6 +147,22 @@ public void checkPermissionForUser2()
         searchQueryService.expectResultsFromQuery(query, user2, FILE_2_NAME, USER_2_FILE_NAME);
     }
 
+    @TestRail (description = "Check that exact term search works. Needs exact term search to be enabled to pass.", section = TestGroup.SEARCH, executionType = ExecutionType.REGRESSION)
+    @Test (groups = TestGroup.SEARCH)
+    public void matchDocumentName()
+    {
+        SearchRequest query = req("cmis", "SELECT * FROM cmis:document WHERE cmis:name = '" + FILE_0_NAME + "' AND CONTAINS('*')");
```

Comment:
Do we really want the CONTAIN('*') here ? 

Wouldn't it be better to disable the option to go via DB (TMDQ) when starting repo ? At least for Elasticsearch CMIS Tests (and possibly AFTS ones as well) ?

Note: for now it requires the following property name (until we look to remove "solr." part):

`-Dsolr.query.cmis.queryConsistency=EVENTUAL`

And remove &/or refactor any tests that do not yet work, eg. basicCMISQueryAgainstDB(). See for example:

https://github.com/Alfresco/alfresco-enterprise-repo/pull/809/files#diff-2466e672e08f8f55c9f5273d5f2a5555f148ce057aaf137da65bbd7df917f35cR41-L44

Reply:
This sounds like a great improvement to these tests, but let's do it as a separate ticket.

Reply:
Hopefully just a couple of lines (assuming no other fallout). But yes, it could also be done as part of ACS-2920 (if not sooner).